client/src/app/core/services/pdf-document.service.ts

```diff
@@ -222,6 +201,7 @@ export class PdfDocumentService {
                 fit: [180, 40],
                 width: '20%'
             });
+            this.images.push({ placeholder: logoHeaderLeftUrl, url: logoHeaderLeftUrl });
```

Comment:
instead of a placeholder, we can calculate the filename by the path.
This would allow us to remove redundant entries before pushing them into VFS